applications/Parallel.md

```diff
@@ -175,17 +175,16 @@ The major deliverable of for this milestone:
-|    3.a | Integration with front-end          | We will integrate our existing front end to the finalized substrate backend                                                                                                                                                                                                                         |
-|    3.b | Article/Tutorial                    | We will create an article and a video demo that explains how users can start using the platform for staking for DOT and KSM. Additionally, we will also create tutorials to walk show the users how they can earn "double interests" through staking and lending at the same time via xDOT and xKSM |
-|    3.c | Docker                              | We will provide a dockerfile to demonstrate the end-to-end use case of Staking/Unstaking DOT and KSM tokens                                                                                                                                                                                         |
-|     4. | User Testing                        | We will conduct user testing to improve our product's UX and UI, to ensure that the borrowing and lending functionalities are intuitive. Initially, we will conduct qualitative user testing by observing 10-15 users use the v1 platform and provide a summary of the findings and improvements made based on insights                                                                                                                |
+|    2.a | Validator evaluation schema         | We will design the validator evaluation schema to select the outstanding validators.                                                                                                                         |
+|    2.b | Offchain client                     | We will set up multi-signature account on relaychain, and operate the `bond` and `unbond` dispatchables by using offchain client.                                                |
+|    3. | Integration with front-end          | We will integrate our existing front end to the finalized substrate backend                                                                                                                                                                                                                         |
+|    4. | Article/Tutorial                    | We will create an article and a video demo that explains how users can start using the platform for staking for DOT and KSM. Additionally, we will also create tutorials to walk show the users how they can earn "double interests" through staking and lending at the same time via xDOT and xKSM |
```

Comment:
We will create an article and a demo video that will explain how users can start using the platform for staking DOT and KSM. Additionally, we will also create tutorials for users to explain how they can earn "double interests" through staking and lending at the same time via xDOT and xKSM.